.vscode/tasks.json

```diff
@@ -37,6 +37,19 @@
                 "/consoleloggerparameters:NoSummary"
             ],
             "problemMatcher": "$msCompile"
+        },
```

Comment:
Different indent from my previous commit. Shall we add any config to enfore the format of vscode settings?

Reply:
Let's follow your indentions for now.